docs/md/flows.md

```diff
@@ -214,16 +214,47 @@ through `extra` on the result.
         "step_uuid": "644592ee-11ad-4bc4-9566-6fb2598c32d6",
         "resthook": "new-registration",
         "payload": {
+            "channel": {
```

Comment:
a lot of these tests are sensitive to whitespace or ordering changes so i added a new one in master which isn't and you can see it isn't affected by this change https://github.com/nyaruka/goflow/pull/678/files#diff-20e5eb1c65c375c018ca6cb5bb957efcL147